### ISSUE TITLE
Legacy Update failed to start in Windows 7 SP1

### ISSUE THREAD

**Issue by myslivets** [kirb/LegacyUpdate#25]
After installing Legacy Update 1.3 from official site in Windows 7 SP1, this program isn't open WU V6 site.

_OS: Windows 7 Ultimate SP1, Ru
IE: 9_

![изображение](https://user-images.githubusercontent.com/60743585/204744285-03cc03f9-693f-498e-ba2c-aab0234de1b5.png)

![изображение](https://user-images.githubusercontent.com/60743585/204744713-39ae1db5-3d15-4f5f-bfb8-d5ae7c2b0250.png)

![изображение](https://user-images.githubusercontent.com/60743585/204748535-f0212fe9-5e7f-4ce4-9c51-9939fe336882.png)

**Comment by kirb** [comment]
Not sure why you’re having trouble loading the site, but be aware that IE9+ aren’t supported at the moment, see #19 for more info. You can use the built-in Windows Update control panel on Vista/7 instead, that will work the same as the website.

**Comment by atauenis** [comment]
Try use IE7 compatibility mode (can be enabled via F12 tools).

**Comment by kirb** [comment]
I fixed this in two ways:

* For some reason when I first tried to set the update site to use IE8 compatibility mode, it wasn't working. I tried again a few weeks ago and it did. (It may have had something to do with me not setting the `Vary:` header for correct caching.)  So it should work in IE9-11 now.
* If that fix happens to not work, I've added an IE version check that will make the site suggest that you use the Windows Update Control Panel instead.
